energy.md

```diff
@@ -45,10 +45,15 @@ GCAM models depletable resources (oil, unconventional oil, natural gas, coal, an
 <img src="gcam-figs/gas_supply_curve.png" width="750" height="300" />
 {: .fig}
 <img src="gcam-figs/coal_supply_curve.png" width="750" height="300" />
-**Illustrative examples of supply curves and totall supplies for fossil resources in GCAM (from GCAM 5.1.2)**
+**Illustrative examples of supply curves and totall supplies for fossil resources in GCAM (from GCAM 5.3)**
 {: .fig}
 
 
+#### Representation of unconventional oil as a technology within crude oil
+
+Unconventional oil and crude oil into 1 resource (crude oil). Note that the ability for countries to produce unconventional oil is maintained by adding it as a subresource and technology within crude oil. This structure allows GCAM to load all costs related to unconventional oil production directly as technology costs. Since unconventional oil is not a resource, unconventional oil now uses crude oil's global resource price.   
```

Comment:
> Unconventional oil and crude oil into 1 resource (crude oil).

Maybe better worded as:
Unconventional oil and crude are combined into a single resource (crude oil).

> to load all costs

Add to this "to load all costs and energy inputs"

> Since unconventional oil is not a resource, unconventional oil now uses crude oil's global resource price.

I think this is a bit misleading.  I would say "Since unconventional oil is combined with crude oil they share a single price that governs supply expansion."  or something like that.

Reply:
I have added the edits. Thanks very much!